packages/lib/determineProfileDir.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Be sure to not commit auto-generated files :) . (Doing so can be easy to do by accident when switching branches/commits).

Reply:
Alright, got it!